examples/config.yaml

```diff
@@ -37,6 +37,10 @@ providers:
   aspsms:
     username: 'username'
     password: 'password'
+  sipgate:
+    username: 'username'
+    password: 'password'
+    url: "https://api.sipgate.com/v2/sessions/sms"
```

Comment:
Is it likely the URL will change? Otherwise I think it'd be nice to simply have it within the code as a constant.

Reply:
I think if the URL changes, it's likely that the payload format would most likely also need changing, so I'm totally ok with hardcoding it if you are.